views/pages/marriage-results.html

```diff
@@ -32,7 +32,7 @@
     {{#records}}
     <li>
       <article>
-        <a href="/details/{{id}}?{{querystring}}&amp;multipleResults">
+        <a href="/marriage/details/{{id}}?{{querystring}}&amp;multipleResults">
```

Comment:
Does this mean that the old `/details` path is no more, and that it is now `/birth/details`?

Reply:
Not yet. Though perhaps in the future.